### PR TITLE
Feature/login ui

### PR DESCRIPTION
**Pull request by ragasudhaa**
create Event form component functionality included.

### CODE REVIEW

File: packages/ui/src/components/createEvent.js [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
+import { Route } from 'react-router-dom';
+import History from '../history';
+import { INVALID_USER } from "../constants/actions";
+
+const participants = [
+  <MenuItem key={1} value={1} primaryText="Organizer" />,
+  <MenuItem key={2} value={2} primaryText="Volunteer" />
```
Comment (shalinijha92) [inline packages/ui/src/components/createEvent.js]: Don't hardcode the values here, get just the primaryText as an array from constants and inside a loop render it.
Participants = ['Organizer', 'Volunteer'] => This should come from constant

renderParticipants = () => {
Participants.map((participant, index) => {
return <MenuItem key={index} value={index} primaryText={participant} />
})
}

Something of this sort

File: packages/ui/src/components/createEvent.js [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+];
+const hourly = [
+  <MenuItem key={1} value={1} insetChildren={true} primaryText="Oliver Hansen" />,
+  <MenuItem key={2} value={2} insetChildren={true} primaryText="Van Henry" />,
+  <MenuItem key={3} value={3} insetChildren={true} primaryText="April Tucker" />,
+  <MenuItem key={4} value={4} insetChildren={true} primaryText="Ralph Hubbard" />
+];
+const nonhourly = [
+  <MenuItem key={1} value={1} insetChildren={true} primaryText="Oliver" />,
```
Comment (shalinijha92) [inline packages/ui/src/components/createEvent.js]: Same as above

File: packages/ui/src/components/createEvent.js [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+        event.preventDefault();
+        if (this.handleValidation()) {
+            this.props.sendEventDetails(this.state.createEventformData,this.props.userInfo)
+            .then((response,error) =>{
+                History.push("/dashboard");
+            },(error)=>{
+              this.handleOpen();
+              this.setState({
+                  open: true,
+                  message:"Event not created!!"
```
Comment (shalinijha92) [inline packages/ui/src/components/createEvent.js]: Put all the messages in a constant and import from there.

File: packages/ui/src/components/createEvent.js (hunk 1 above)
Comment (ragasudhaa) [reply packages/ui/src/components/createEvent.js]: These values will be removed once the API is ready. Madhulika is working on it

**Pull request merged**